code/style.css

```diff
@@ -1,28 +1,32 @@
 /* Global css variables used for colors */
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: #6f896e;
+  --secondary: #d8c172;
 }
 
 body {
   display: flex;
+  background-color:#272739;
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
+  font-family:'Bebas Neue';
+
 }
 
 h1 {
-  font-size: 42px;
+  font-size: 64px;
   font-weight: 500;
   line-height: 48px;
   margin: 10px 0;
   color: white;
+  font-family:'Bebas Neue';
```

Comment:
Hmm.. do you need to specify the font-family when it's already being stated in the body { } ?